views/partials/select-server-header.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="selectServer__title">
```

Comment:
are these going to be re-used in the future?

Reply:
Yes, there is another screen similar to this one, i.e. the list of GH apps. It can be re-used there for [ticket](https://softwareteams.atlassian.net/browse/ARC-1437): 
There could be more in the future too.